src/util.c

```diff
@@ -0,0 +1,47 @@
+#include "jemalloc/internal/util.h"
```

Comment:
Always include these two plus a blank line at the beginning:
```
#include "jemalloc/internal/jemalloc_preamble.h"
#include "jemalloc/internal/jemalloc_internal_includes.h"
```
See example: https://github.com/jemalloc/jemalloc/blob/dev/src/bin_info.c#L1-L3